sentry_sdk/integrations/huey.py

```diff
@@ -124,8 +120,7 @@ def _wrap_task_execute(func):
     # type: (F) -> F
     def _sentry_execute(*args, **kwargs):
         # type: (*Any, **Any) -> Any
-        hub = Hub.current
-        if hub.get_integration(HueyIntegration) is None:
+        if sentry_sdk.get_client().get_integration(HueyIntegration) is None:
```

Comment:
I think we can use `ensure_integration_enabled` here. Is there a reason why we cannot use it?